Xamarin.PropertyEditing.Mac/Controls/Custom/BasePopOverViewModelControl.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class BasePopOverViewModelControl : BasePopOverControl
+	{
+		internal IEnumerable<object> Targets => this.ViewModel.Editors.Select (ed => ed.Target);
```

Comment:
Doesn't seem that we're using this anywhere?

Reply:
IIRC this would be used in the RequestResource, but I'll add that in that PR.